### ISSUE TITLE
Option: replace toolbox with distrobox

### ISSUE THREAD

**Issue by trytomakeyouprivate** [iaacornus/Fedora-OSTree-Setup#34]
They work exactly the same, but toolbox only supports Fedora (being only useful for Fedora devs), while Distrobox supports every linux distro, so that you can use any .deb, AUR package e.g. natively, even create appstarters

```
# remove
toolbox

# add to install RPMs
distrobox
```
only problem: Distrobox uses Docker by default.

**Comment by iaacornus** [comment]
> only problem: Distrobox uses Docker by default.

doesnt sounds much of a problem to me

**Comment by trytomakeyouprivate** [comment]
afaik Docker is ran by Microsoft but anyways, it uses Podman it seems, even though Docker containers are supported?

Toolbox now also has support for more Distros, but I would still keep this choice. Distrobox runs exactly the same, even uses the same commands, and has much more functionality. It even creates nice little appstarters to run the console in that box, and you can export app icons too, so they run and feel as if they were native.

**Comment by iaacornus** [comment]
> afaik Docker is ran by Microsoft but anyways, it uses Podman it seems, even though Docker containers are supported?
> 
> Toolbox now also has support for more Distros, but I would still keep this choice. Distrobox runs exactly the same, even uses the same commands, and has much more functionality. It even creates nice little appstarters to run the console in that box, and you can export app icons too, so they run and feel as if they were native.

yeah distrobox has more features, ill keep it in options that users can select
